cmd/agent/app/sampling/tcollector_proxy.go

```diff
@@ -42,8 +42,8 @@ type tcollectorSamplingProxy struct {
 }
 
 // NewTCollectorSamplingManagerProxy implements Manager by proxying the requests to tcollector.
-func NewTCollectorSamplingManagerProxy(channel *tchannel.Channel, mFactory metrics.Factory, clientOpts *thrift.ClientOptions) Manager {
-	thriftClient := thrift.NewClient(channel, "tcollector", clientOpts)
+func NewTCollectorSamplingManagerProxy(svc string, channel *tchannel.Channel, mFactory metrics.Factory, clientOpts *thrift.ClientOptions) Manager {
```

Comment:
CollectorProxy (struct/file) might be a better name (sampling implied, tcollector is irrelevant)